src/frequenz/sdk/timeseries/_moving_window.py

```diff
@@ -9,7 +9,7 @@
 import math
 from collections.abc import Sequence
 from datetime import datetime, timedelta
-from typing import SupportsIndex, overload
+from typing import SupportsIndex, assert_never, overload
```

Comment:
assert never doesn't seem to be used? at least not in the commit where it is added

Reply:
Done.